packaging/ubuntu/debian/control

```diff
@@ -34,6 +34,7 @@ Build-Depends: asciidoc,
                qtpositioning5-dev,
                qml-module-qtpositioning,
                qml-module-qtlocation,
+	       qml-module-qtquick2,
```

Comment:
hehe - whitespace... I keep doing that one wrong myself. For some reason this is one of the very few files with eight spaces instead of a tab - I actually think tab would work and we could just convert it. Anyway, nothing to worry about, I'll pull as is.
Thanks, @mturkia